patchwork-events-entity/src/main/java/com/patchworkmc/impl/event/entity/EntityEvents.java

```diff
@@ -54,10 +57,19 @@ public static boolean onLivingDeath(LivingEntity entity, DamageSource src) {
 		return MinecraftForge.EVENT_BUS.post(new LivingDeathEvent(entity, src));
 	}
 
+	public static boolean onLivingUpdateEvent(LivingEntity entity) {
+		return MinecraftForge.EVENT_BUS.post(new LivingEvent.LivingUpdateEvent(entity));
+	}
+
 	public static boolean onEntityJoinWorld(Entity entity, World world) {
 		return MinecraftForge.EVENT_BUS.post(new EntityJoinWorldEvent(entity, world));
 	}
 
+	// PlayerEvents
+	public static boolean onPlayerLoggedIn(ServerPlayerEntity playerEntity) {
```

Comment:
Probably don't need to return a `boolean` here since the event is not cancellable.